src/Form/Extension/FormTypeExtension.php

```diff
@@ -118,12 +118,8 @@ public function finishView(FormView $view, FormInterface $form, array $options)
 
     public function setDefaultOptions(OptionsResolverInterface $resolver)
     {
-        $resolver->setDefaults(array(
-            'jquery_validation' => $this->defaultEnabled,
-        ));
-        $resolver->setAllowedTypes(array(
-            'jquery_validation' => array('bool', 'null'),
-        ));
+        $resolver->setDefault('jquery_validation',$this->defaultEnabled);
+        $resolver->setAllowedTypes('jquery_validation',array('bool','null'));
     }
 
```

Comment:
Instead of changing the `setDefaultOptions` method maybe it would be better to add the `configureOptions` method that will replace `setDefaultOptions` in Symfony 3.0.

This would add some duplicate code but I'm fine with that if you add `@deprecated since 1.1 due to symfony deprecation in version 2.7.` to the `setDefaultOptions` method.